packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -1357,6 +1373,28 @@ async function loadBundlerConfig(
 
   invariant(conf?.contents != null);
 
+  // minBundles will be ignored if shared bundles are disabled
```

Comment:
Good warning, but its also important to note that "minBundleSize" also only applies to shared bundles. Should we have a warning for that as well?